include/gmsh.h

```diff
@@ -34,10 +35,7 @@ class GMSH : public Mesh<Tdim, Tvertices> {
   void store_element_vertices();
 
   //! Compute material points from element coordinate map
-  void compute_material_points();
-
-  //! Compute stresses
-  void compute_stresses();
+  void compute_material_points(const double density, const double k0);
```

Comment:
Why are parameters `density` and `k0` passed to mesh? This should be in material properties.